src/log.ts

```diff
@@ -175,6 +175,10 @@ export namespace message {
     return `${channel} dropped as it is incompatible with "${markOrFacet}"${when ? ` when ${when}` : ''}.`;
   }
 
+  export function invalidEncodingChannel(encoding: string) {
```

Comment:
`encoding` normally refer to encoding mapping, which is an object.

Rename to `channel` would be less misleading.

Reply:
Done.